lib/connection-parameters.js

```diff
@@ -82,6 +83,9 @@ ConnectionParameters.prototype.getLibpqConnectionString = function(cb) {
   if(this.database) {
     params.push("dbname='" + this.database + "'");
   }
+  if(this.replication) {
+    params.push("replication='" + (this.database === true ? "true" : this.replication) + "'");
```

Comment:
I implemented pg_recvlogical's access method the same way.
The replication value can have a value of true or database.

https://github.com/pipelinedb/pipelinedb/blob/bdcd1fcd37584a3ad5427bb1a1b27a1af7521f0c/src/bin/pg_basebackup/streamutil.c#L105